app/locales/en/english.json

```diff
@@ -32,10 +32,15 @@
 
 
 
-    "import_title": "Import Locations",
-    "import_step_1": "1. Login to your Google Account and Download your Location History",
-    "import_step_2": "2. After downloaded, open this screen again. The data will import automatically.",
-
+    "import_title":"Import Locations",
+    "import_step_1":"Adding location data from Google will give you a head start on building your recent locations.",
+    "import_step_2":"Before you can import, you must first 'Take out' your location data from Google.",
+    "import_step_3":"Visit Google Takeout and export your Location History using following settings: \n1. Delivery method: \"Google Drive\" \n2. Frequency: \"Export once\" \n3. File type & size: \".zip\" and \"1GB\" \n4. Return here to import the archive",
```

Comment:
I think there is an extra step of "Google sends you an email to tell you the export is ready", right?